Tests/Tests/CoreData/TracksTestContextManager.swift

```diff
@@ -0,0 +1,60 @@
+@testable import AutomatticTracks
+import Foundation
+
+fileprivate var persistentStoreExistsKey: UInt8 = 0
+
+/// This class duplicates the class found in `TestTracksContextManager.swift`, because there's no support in
+/// packages for mixed-language source files and method-calling.  We could work around this problem
+/// in other ways but since this will eventually move to Swift, I may as well just duplicate it for now and we
+/// can clean-up once we remove the remaining ObjC code.
+///
+class TracksTestContextManager: TracksContextManager {
+
+    private var persistentStoreExists: Bool {
+        get {
+            guard let value = objc_getAssociatedObject(self, &persistentStoreExistsKey) as? Bool else {
```

Comment:
Why does this need to use associated objects instead of just declaring properties directly?

Reply:
Changed to a `private var`.